schema/url/plots.raml

```diff
@@ -24,6 +24,24 @@ types:
         minItems: 1
         required: true
 
+  PieplotData:
+    additionalProperties: false
+    properties:
+      overlayVariableDetails:
+        type: StrataVariableDetails
+      facetVariableDetails:
+        type: StrataVariableDetails[]
+        maxItems: 2
+        required: false
+      value:
+        type: number[]
+        minItems: 1
+        required: true
+      label:
```

Comment:
Not sure if `label` is necessary? Would this be duplicated in the overlayVariableDetails?